addons/website_sale/views/templates.xml

```diff
@@ -201,7 +202,7 @@
         <t t-call="website.layout">
             <t t-set="additional_title">Shop</t>
             <div id="wrap" class="js_sale">
-                <div class="oe_structure" id="oe_structure_website_sale_products_1"/>
+                <div class="oe_structure oe_structure_small" id="oe_structure_website_sale_products_1"/>
```

Comment:
I removed this from the specs, it is the task of RVE and it's more that just smaller, best to avoid useless conflicts.

Reply:
Yeah sbu told me and it's in a separate commit, I just need to drop it.